azkaban-common/src/test/java/azkaban/server/HttpRequestUtilsTest.java

```diff
@@ -161,11 +161,12 @@ public void testGetJsonBodyForListOfMapObject() throws IOException, ServletExcep
   @Test
   public void testGetJsonBodyForSingleMapObject() throws IOException, ServletException {
     final HttpServletRequest httpRequest = Mockito.mock(HttpServletRequest.class);
-    final String originalString = "  {\n" + "    \"action\": \"update\",\n" + "    \"table\": \"ramp\",\n"
-        + "    \"conditions\" : {\n" + "      \"rampId\" : \"dali\"\n" + "    },\n"
-        + "    \"values\": {\n"
-        + "      \"rampStage\": 2,\n" + "      \"lastUpdatedTime\": 1566259437000\n" + "    }\n"
-        + "  }\n";
+    final String originalString =
```

Comment:
I would suggest putting this in the resources directory with the same package as of this test class. Then you can utilize the method
azkaban.utils.TestUtils#readResource to read it as string.

Reply:
> I would suggest putting this in the resources directory

Ok, I'll do that. But as you can see this wasn't added as part of this PR. Formatting changed by Save Actions.